### PR TITLE
MBS-7550 test minimized test-app with help of slackhq/keeper [MERGE_WHEN_READY]

### PR DESCRIPTION
**Pull request by dsvoronin**

### CODE REVIEW

File: subprojects/android-test/test-app/build.gradle.kts [hunk 1]
```diff
@@ -52,6 +54,22 @@ android {
...
 
+    buildTypes {
+        register("staging") {
+
+            initWith(named("debug").get())
+
+            setMatchingFallbacks("debug")
+
+            isMinifyEnabled = true
+            isShrinkResources = false
```
Comment (dsvoronin) [inline subprojects/android-test/test-app/build.gradle.kts:65]: `java.lang.ClassNotFoundException: Didn't find class "android.view.x" on path: DexPathList`
on shrink enabled, haven't dig in it yet

File: docs/content/test/TestMinimized.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+type: docs
+---
+
+# Testing minimized build
+
+About minimization: [link]({{< ref "/assemble/Minimization.md" >}})
+
+We want to run our ui tests against build as close as possible to production one.\
+It's quite a challenging task, mostly because of [tooling problems](https://issuetracker.google.com/issues/126429384)\
+Developers should maintain list of keep rules of code referenced from test app manually.
```
Comment (eugene-krivobokov) [inline docs/content/test/TestMinimized.md]: a list

File: docs/content/test/TestMinimized.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+We use [keeper](https://slackhq.github.io/keeper/) to workaround these problems.
```
Comment (eugene-krivobokov) [inline docs/content/test/TestMinimized.md]: "You can **work around** this problem using a **workaround**."

https://www.macmillandictionary.com/dictionary/british/work-around

File: docs/content/test/TestMinimized.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Build type
+
+Out build types: [link]({{< ref "/assemble/BuildTypes.md" >}})
```
Comment (eugene-krivobokov) [inline docs/content/test/TestMinimized.md]: Out -> Our ?

File: docs/content/test/TestMinimized.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+We chose `staging` as a type to test against for now, it is based on debug, but with minimization and resource shrinking enabled.\
+Staging used as main type for [manual testing]({{< ref "/test/Manual.md" >}}), as it offers all debug options.\
```
Comment (eugene-krivobokov) [inline docs/content/test/TestMinimized.md]: staging -> staging build type ?

File: docs/content/test/TestMinimized.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+However we could miss bugs related to debug, not release sources included in build, so we should reconsider and introduce 4th type in the future.
```
Comment (eugene-krivobokov) [inline docs/content/test/TestMinimized.md]: not release -> non-release ?

**Review by eugene-krivobokov — APPROVED** [review]

**Pull request merged**